Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.13.2-alpine
+
+WORKDIR /leqo-backend
+
+COPY ./pyproject.toml /leqo-backend/pyproject.toml
+
+COPY ./.python-version /leqo-backend/.python-version
+
+COPY ./uv.lock /leqo-backend/uv.lock
+
+RUN pip install uv
+
+RUN pip install "fastapi[standard]"
```

Comment:
We are using `uv` instead of `pip` as our package manager.

You can have a look at the uv docs for a simple docker sample:
https://docs.astral.sh/uv/guides/integration/fastapi/#deployment